airrohr-firmware/airrohr-firmware.ino

```diff
@@ -3946,13 +3945,14 @@ void loop() {
 
 		server.begin();
 
+		const unsigned long ONE_DAY_IN_MS = 24 * 60 * 60 * 1000;
 		//forced restart after 4 weeks
-		if ((act_milli - last_update_attempt) > (28 * pause_between_update_attempts)) {
+		if ((act_milli - time_point_device_start_ms) > (28 * ONE_DAY_IN_MS)) {
 			ESP.restart();
 		}
 
 		// check for updates (default once per day)
-		if ((act_milli - last_update_attempt) > pause_between_update_attempts) {
+		if ((act_milli - last_update_attempt) > ONE_DAY_IN_MS) {
```

Comment:
This is not necessary one day in the future. If there are more and more sensors we may change this to another timespan. Thats why this variable was called pause_between update_attempts.